flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateCheckpointWriterTest.java

```diff
@@ -218,12 +221,14 @@ public void testRecordingOffsets() throws Exception {
         for (InputChannelStateHandle handle : result.inputChannelStateHandles.get()) {
             int headerSize = Integer.BYTES;
             int lengthSize = Integer.BYTES;
-            assertEquals(singletonList((long) headerSize), handle.getOffsets());
-            assertEquals(
-                    headerSize + lengthSize + numBytes * offsetCounts.remove(handle.getInfo()),
-                    handle.getDelegate().getStateSize());
+            assertThat(handle.getOffsets()).isEqualTo(singletonList((long) headerSize));
+            assertThat(handle.getDelegate().getStateSize())
+                    .isEqualTo(
+                            headerSize
+                                    + lengthSize
+                                    + numBytes * offsetCounts.remove(handle.getInfo()));
         }
-        assertTrue(offsetCounts.isEmpty());
+        assertThat(offsetCounts.isEmpty()).isTrue();
```

Comment:
```suggestion
        assertThat(offsetCounts).isEmpty();
```
assertJ has `isEmpty()`